runner_facebook.sh

```diff
@@ -1,23 +1,25 @@
 #!/usr/bin/env bash
 
-OLD=$PWD
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
-
 cd $DIR
 
-if [ -d "/usr/local/crawler-venv" ]
-then
-  source /usr/local/crawler-venv/bin/activate
-fi
-
-FOLDER=$(date +"%Y-%m")
-mkdir -p "$OLD/$FOLDER"
```

Comment:
Removing this and using the other variables to create the folders make more sense